test/lib/jdk/test/lib/Platform.java

```diff
@@ -382,14 +384,13 @@ private static String variant() {
 
 
     public static boolean isDefaultCDSArchiveSupported() {
-        return (is64bit()  &&
-                isServer() &&
-                (isLinux()   ||
-                 isOSX()     ||
-                 isWindows()) &&
-                !isZero()    &&
-                !isMinimal() &&
-                !isARM());
+        return is64bit()
+               && isServer()
+               && (isLinux() || isOSX() || isWindows())
+               && !isZero()
+               && !isMinimal()
+               && !isARM()
+               && !isJ9();
```

Comment:
I'd like to see this change reduced. I suggest just one line needs to change:
```
    isServer() &&
```
to
```
    isServer() && !isJ9() &&
```

Reply:
Updated.